producer-c/producer-cloudwatch-integ/KvsProducerSampleCloudwatch.cpp

```diff
@@ -180,26 +197,53 @@ INT32 main(INT32 argc, CHAR *argv[])
             frame.presentationTs = frame.decodingTs;
             frameIndex++;
         }
+        if (STATUS_FAILED(retStatus)) {
+            DLOGE("Failed with status 0x%08x\n", retStatus);
+         }
 
+        if (frame.frameData != NULL) {
+            MEMFREE(frame.frameData);
```

Comment:
When there is more processing after freeing, I always recommend NULL-ing it to make sure there is no duplicate free. Instead of if(not null) free; set to null; sequence, you could simply do SAFE_MEMFREE(fame.frameData);

Reply:
Makes sense. Fixed.